docs/manual/forms/logic_dmn.rst

```diff
@@ -0,0 +1,55 @@
+.. _manual_logic_dmn:
+
+====================
+Beslistabellen (DMN)
+====================
+
+Open Formulieren heeft een module om beslistabellen/beslissingen te evalueren. Deze
+dient :ref:`geconfigureerd <configuration_dmn_index>` te zijn voor je hier gebruik van
+kan maken.
+
+.. seealso:: Zie ook het voorbeeldformulier :ref:`examples_camunda`.
+
+Wat zijn beslistabellen?
+========================
+
+In beslistabellen kan je regels vastleggen - op basis van een aantal inputs en deze
+regels volgt hier een (berekend) resultaat uit. Deze tabellen worden door een
+*DMN-engine* uitgevoerd om het resultaat te bepalen. Beslistabellen zijn doorgaans
+een overzichtelijke, gecentraliseerde manier om regels (zoals wetgeving) te toetsen,
+en zijn vaak een goed alternatief voor complexe JsonLogic-expressies.
+
+Configuratie
+============
+
+DMN-acties moeten geconfigureerd worden.
+
+**Algemene instellingen**
+
+* Pluginkeuze
+* Beslisdefinitie/tabel om te evalueren
+* Eventuele versie van de beslissing - indien niet opgegeven, dan wordt de meest recente
+  versie gebruikt.
+
+**In- en uitvoerparameters**
+
+Bij de invoerparameters wordt de waarde van formuliervariabelen ingestuurd als input
+voor de geselecteerde beslistabel. Formuliervariabelen en beslistabelvariabelen kunnen
+hierbij verschillende namen hebben.
+
+Op deze manier stuur je de gebruikersinvoer tijdens het invullen van het formulier naar
+de beslistabel. Je kan ook statische variabelen of zelf-gedefinieerde variabelen koppelen.
+
+.. note::
+
+   Let op dat je het juiste datatype selecteert - als de tabel een getal verwacht,
+   dan mag je geen tekstuele waarden/variabelen koppelen.
+
+   Als je een letterlijke booleanwaarde (``true``/``false``) nodig hebt, let er dan op
+   dat je als formulierveld een selectievakje gebruikt - radio en keuzemenu-opties
+   worden namelijk als tekstuele waarde geïnterpreteerd.
```

Comment:
Maybe this should be a warning? I think that attracts more attention than a note, then again I'm not sure how often this is used/going to be used by municipalities

Reply:
I wasn't sure whether it should be info or warning. I feel that warning is much more serious in the sense that "if you don't follow this advice, you will likely break something or get unexpected results", while the idea of this documentation is more to cover our ***: "we know this limitation exists".